### PR TITLE
feat:update clonemodal and releeasehistorymodal css

### PR DESCRIPTION
**Pull request by kogel-net**
# Description

_Please explain the changes you've made_

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

**Comment by doddgu** [comment]
@codding-y PTAL

File: src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Rcl/Pages/Modal/CloneModal.razor [hunk 1]
```diff
@@ -54,7 +54,7 @@
                                                              ValueChanged="projectId=>ProjectValueChangedAsync(projectId)"
                                                              Label="@T("Project")"
                                                              MenuProps="props => props.OffsetY = true"
-                                                             Outlined
+                                                                          Outlined
```
Comment (codding-y) [inline src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Rcl/Pages/Modal/CloneModal.razor:57]: remove black space

File: src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Rcl/Pages/Modal/CloneModal.razor.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿
+::deep #d-clone-modal {
+    min-height: 48px !important;
+    background-color: red;
```
Comment (codding-y) [inline src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Rcl/Pages/Modal/CloneModal.razor.css:4]: red?

File: src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Rcl/Pages/Modal/CloneModal.razor.css [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+}
+
+    ::deep #d-clone-modal .m-list-item {
+        min-height: 48px !important;
+        background-color: red;
```
Comment (codding-y) [inline src/Web/Masa.Dcc.Web.Admin/Masa.Dcc.Web.Admin.Rcl/Pages/Modal/CloneModal.razor.css:9]: The design is red？

**Review by codding-y — CHANGES_REQUESTED** [review]
Razor code format

**Review by codding-y — APPROVED** [review]

**Review by codding-y — APPROVED** [review]
LGTM

**Comment by doddgu** [comment]
@kogel-net resolve conflict

**Pull request merged**